### ISSUE TITLE
Include option to disable changing background-color of current text in editor

### ISSUE THREAD

**Issue by fduniho** [terrylinooo/githuber-md#74]
**Any issue will be ignored if you don't provide the following detailed information.**

**Describe the bug**

It's not a bug, but for several themes, it's incredibly annoying that the background color of the current paragraph in the editor pane changes to a different color. Please include the option to disable this.

**To reproduce**

1. Edit a Wordpress post while WP Githuber MD is running.
2. Move the cursor around the HTML or Markdown code in the left pane.
3. Notice that the background color for the current paragraph is different than for surrounding paragraphs.
4. Try out different dark themes, and notice that the background color used for this is generally less suitable than the theme's normal background color.

**Expected behavior**

The Markdown tab in the settings should include an option for keeping the background-color the same as surrounding text for current text in the editor.

**Screenshots**

**Server environment**

 - WordPress version 5.2
 - WP Githuber MD plugin version 1.11.3
 - PHP version 7.2.17

**User environment**

 - OS: Windows 7
 - Browser: Vivaldi, Firefox

**Additional context**

**Comment by terrylinooo** [comment]
Okay.

**Comment by fduniho** [comment]
I see that there is now a new option called "Match Highlighter". Based on its description, it might be intended to be what I suggested. But it doesn't seem to do anything. To illustrate what I had in mind, I'll include a screenshot and a recolored modification of it. The screenshot, called screenshot.png, shows what editing one of my pages looks like with the blackboard theme. The cursor is in the third paragraph, and because of this, it has a brown background instead of a black background. Changing the value of Match Highlighter has no effect on this. The recolored.png file is a modification of the screen shot. I have changed the background color of the third paragraph to match the background color of the rest of the document. This is what I had in mind with my suggestion.
    On Wednesday, May 22, 2019, 9:22:33 PM EDT, Terry L. <notifications@github.com> wrote:  
 
 
Closed #74.
